circuits/circom/core/qv/pollJoining.circom

```diff
@@ -0,0 +1,85 @@
+pragma circom 2.0.0;
+
+
+// circomlib import
+include "./mux1.circom";
+// zk-kit imports
+include "./safe-comparators.circom";
+// local imports
+include "../../utils/hashers.circom";
+include "../../utils/messageToCommand.circom";
```

Comment:
there's quite a few imports we don't need:
messageToCommand - processMesagesInputHasher - stateLeafAndBallotTRansformer - incrementalQuinTree

Reply:
Done